pkgs/os-specific/linux/asus-wmi-sensors/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchFromGitHub, kernel }:
+
+stdenv.mkDerivation rec {
+  name = "asus-wmi-sensors-${version}-${kernel.version}";
+  version = "0.0.1";
+
+  # The original was deleted from github, but this seems to be an active fork
+  src = fetchFromGitHub {
+    owner = "electrified";
+    repo = "asus-wmi-sensors";
+    rev = "8daafd45d1b860cf5b17eee1c94d93feb04164a9";
+    sha256 = "0kc0xlrsmf783ln5bqyj6qxzmrhdxdfdd2b9ygf2lbl2153i04vc";
+  };
+
+  hardeningDisable = [ "pic" ];
+
+  nativeBuildInputs = kernel.moduleBuildDependencies;
+
+  preConfigure = ''
+    sed -i 's|depmod|#depmod|' Makefile
+  '';
+
+  makeFlags = [
+    "TARGET=${kernel.modDirVersion}"
+    "KERNEL_MODULES=${kernel.dev}/lib/modules/${kernel.modDirVersion}"
+    "MODDESTDIR=$(out)/lib/modules/${kernel.modDirVersion}/kernel/drivers/hwmon"
+  ];
+
+  meta = with stdenv.lib; {
```

Comment:
I think it's missing maintainers list which is mandatory due to the https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md

I'm happy to be the maintainer for that package as I'm using it for my lab and following closely the project.

Reply:
Sorry for many comments, but I've figured out that it'll not work for old kernels, so, we need to have `broken = versionOlder kernel.version "4.12";` in `meta` as https://github.com/electrified/asus-wmi-sensors#how-to-install saying that 4.12 or older is required. After adding it, this change successfully pass the `nixpkgs-review` and produces the following packages: 
```
11 package built:
linuxPackages-libre.asus-wmi-sensors linuxPackages.asus-wmi-sensors linuxPackages_4_14.asus-wmi-sensors linuxPackages_5_4.asus-wmi
-sensors linuxPackages_hardened.asus-wmi-sensors linuxPackages_latest-libre.asus-wmi-sensors linuxPackages_latest_hardened.asus-wm
i-sensors linuxPackages_latest_xen_dom0.asus-wmi-sensors linuxPackages_testing_bcachefs.asus-wmi-sensors linuxPackages_testing_har
dened.asus-wmi-sensors linuxPackages_xen_dom0.asus-wmi-sensors
```

W/o it it fails following way:
```
3 package marked as broken and skipped:
linuxPackages_5_3.asus-wmi-sensors linuxPackages_hardkernel_4_14.asus-wmi-sensors linuxPackages_hardkernel_latest.asus-wmi-sensors

2 package failed to build:
linuxPackages_4_4.asus-wmi-sensors linuxPackages_4_9.asus-wmi-sensors
```

Reply:
Updated the PR with according to your comment. Thanks for catching that.

Reply:
I think other packages using following format for maintainers, so, would be good to use it here as well:

```
maintainers = with maintainers; [ Frostman ];
```

Additionally, master was switched to 5.5 kernel, so, I've verified it against the new kernel as well.

Thanks

Reply:
Done.